tsl/test/src/remote/node_killer.h

```diff
@@ -10,9 +10,18 @@
 #include <libpq-fe.h>
 
 #include "remote/connection.h"
+#include "remote/dist_txn.h"
 
-typedef struct RemoteNodeKiller RemoteNodeKiller;
-extern RemoteNodeKiller *remote_node_killer_init(TSConnection *conn);
+typedef struct RemoteNodeKiller
+{
+	pid_t pid;
+	DistTransactionEvent killevent;
+	const TSConnection *conn;
+	unsigned num_kills;
```

Comment:
Nit: avoid `unsigned`, it can lead to strange wrap-arounds, and you don't need that extra bit. Stick with `int` unless there is a good reason to use the wrap-around logic of `unsigned`.

Reply:
Change, but, FWIW, the wrap-around issue is not possible with current code and I think unsigned properly signals that this cannot be a negative value.